src/__tests__/lib/utils/SynapseClient.test.tsx

```diff
@@ -5,68 +5,48 @@ import {
   FileHandleAssociateType,
   PaginatedResults,
 } from '../../../lib/utils/synapseTypes/'
-import { SynapseClient, SynapseConstants } from '../../../lib/utils/'
+import { SynapseClient } from '../../../lib/utils/'
 import { FunctionReturningPaginatedResults } from '../../../lib/utils/SynapseClient'
 
 describe('it works at integration level testing', () => {
```

Comment:
I don't really like these tests. In general, they seem to be testing the Synapse backend rather than the behavior of the client itself. Would like to eventually replace with MSW, but not right now.